### DISCUSSION TITLE
Multiple screens

### DISCUSSION THREAD

**Discussion by marryverry123** [linearmouse/linearmouse#587] [Q&A]
Hi there, im new Mac user and I've got completely no idea how to use all of this new stuff. Ive installed linear mouse but I've got two displays connected to my Mac Pro m2 and mouse is working absolutely different in the two of them - can I fix it somehow?

**Comment by lujjjh** [comment]
Hey @marryverry123,

Are these two displays in different resolutions? Could you elaborate on how the pointer behaves differently when it's on each of them?

There is also a similar feature request (#460) to add support for display-dependent settings.

**Reply by marryverry123** [reply]
Yes, I've got application which is called better display and all of the preferences for both of them are the same. The pointer is not acting so smoothly in one of them, it kinda move slower, like I set up the lower acceleration, but I was checking - it's still the same

**Reply by lujjjh** [reply]
Perhaps you could check whether there is a difference in the refresh rate of the two displays.  [This webpage](https://testufo.com/) can be used to test the refresh rate in case that you're unsure how to check the refresh rate.

**Reply by marryverry123** [reply]
refresh rate is also the same, but rn I see that maybe display settings are the problem... the second one is showing different resolution even if the app and original Mac settings are showing all the same for both of them
